### PR TITLE
Fixed groupme icon

### PR DESCRIPTION
**Pull request by harmless-tech**
Issue #7

### CODE REVIEW

File: src/components/lib/Label.tsx [hunk 1]
```diff
@@ -46,12 +47,11 @@ export const IconLabel: FC<IconLabelProps> = ({
...
-export const Icon: FC<IconProps> = ({ fillStyle = "", icon = "" }) =>
-    <span
-        className={classNames([
-            "fa" + fillStyle.charAt(0),
-            icon.startsWith("fa-")
-                ? icon
-                : "fa-" + icon
-        ])}
-    />;
+export const Icon: FC<IconProps> = ({ fillStyle = "", icon = "", fa = "true" }) => {
```
Comment (DonIsaac) [inline src/components/lib/Label.tsx]: default to a boolean, noy a string

**Review by DonIsaac — CHANGES_REQUESTED** [review]

File: src/components/lib/Label.tsx (hunk 1 above)
Comment (harmless-tech) [reply src/components/lib/Label.tsx]: Done

**Comment by DonIsaac** [comment]
@harmless-tech what's the status of this PR?

**Comment by harmless-tech** [comment]
> @harmless-tech what's the status of this PR?

This PR is done and ready to be merged.

**Review by DonIsaac — APPROVED** [review]

**Pull request merged**